acceptance_process.md

```diff
@@ -81,25 +54,25 @@ The objective of the Network Steward is to make funding available for projects w
+* Did the applicant declare and control for any potential conflicts of interest ?
 
-## Counter-proposal and Amendment Criteria
-Counter-proposals and amendments to existing proposals are compared using the 5 evaluation criteria but in addition, they must achieve the status of significantly similar and a significant improvement.
-
-### Significantly similar
-A counter-proposal or amendment should be considered to be significantly similar in the event that it targets the same objectives as the original proposal. Some questions which may be useful to the evaluation of significantly similar status include:
-* Do the two proposals have the same short and long term impacts ?
-* Assuming both projects were funded, would it be nonsensical for a single entity to use the results of both projects ?
-
-### A significant improvement
-The definition of a significant improvement is designed to prevent "stealing projects" by submission of proposals which are trivially less costly. While the exact definition of a significant improvement will be left to the discretion of the NS team, applicants can generally consider that:
-* A proposal of otherwise equal value but at more than 5% less cost is a significant improvement
-* A proposal which is not only better than the original, but in fact also better than another project which is better than the original is a significant improvement
-Counter-proposers should be aware that a copy/paste counter-proposal will generally not be considered to be equal to the original without a justification of domain knowledge equal to that which is justified by the original applicant through the text of the proposal.
-
-### Unacceptable projects
+## Unacceptable projects
 Some projects may be judged by the Network Steward to be unacceptable under any conditions. An intuitive example would be a proposal to build an assasination market which is blatantly illegal and contrary to the objectives of the project. These judgements will be made by each member of the NS team individually and projects which are not deemed acceptable by at least 3 of the 5 team members will not be funded, even if that means burning coins. Some questions which are useful in the evaluation of acceptability:
 
 * Does the project seem more likely to benefit the PKT ecosystem than to harm it ?
 * Does the project seem unlikely to cause significant damage to the perceived legitimacy of the Network Steward ?
 * Does the project seem likely to be a better use of resources than burning the coins ? (note that burning coins causes deflation which increases the value of all existing coins)
 
+## Milestone reports
```

Comment:
This section feels a bit unclear and is a really long sentence. Any chance we can reword this?